src/transactions/InflationOpFrame.cpp

```diff
@@ -82,8 +86,8 @@ InflationOpFrame::doApply(Application& app, LedgerDelta& delta,
     lcl.feePool = 0;
     lcl.inflationSeq++;
 
-    // now credit each account
-    innerResult().code(INFLATION_SUCCESS);
+    // now credit each account (landmine)
```

Comment:
```suggestion
    // now credit each account (landmine for disabled inflation)
```

Reply:
I would rather have no comment at all, but at the least I want it to be vague.